src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  rockets: [],
+};
+
+const rocketsSlice = createSlice({
+  name: 'rockets',
+  initialState,
+  reducers: {},
+});
+
+export const rocketsActions = rocketsSlice.actions;
+
+export default rocketsSlice;
```

Comment:
I think what you want to export at the end is "export default rocketsSlice.reducer".The actions you export in line 13. You can change line 15 to export the reducer to the store.

Reply:
@myelin0 
Thank you for this suggestion, and I  have called the rocketsSlice.reducer in the ```index.js``` file when setting up the reducers.